Source/WebCore/rendering/RenderAttachment.h

```diff
@@ -66,9 +65,9 @@ class RenderAttachment final : public RenderReplaced {
     bool m_hasShadowControls { false };
 };
 
-inline RenderAttachment* HTMLAttachmentElement::renderer() const
+inline RenderAttachment* HTMLAttachmentElement::attachmentRenderer() const
 {
-    return downcast<RenderAttachment>(HTMLElement::renderer());
+    return dynamicDowncast<RenderAttachment>(HTMLElement::renderer());
 }
```

Comment:
We should really delete this method, and downcast + assert in the right places. It's misleading since attachmentRenderer isn't always valid.